### DISCUSSION TITLE
2024-05-31 - 🦙 TinaCMS Pitch - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Rules.Content#8626] [Chewing the Fat]
Hey SSWers,

Lets talk about this rule
https://www.ssw.com.au/rules/pitch-a-product

Imagine someone asks you about a cool product like TinaCMS...

Which one of these 4 videos do you like and why?
- Piers' video - Developers perspective
https://www.youtube.com/watch?v=7yQ1dEOI00M

- Gert's video - Business persons perspective
https://www.youtube.com/watch?v=qGDmO0RPHlI

- Adam's video - History perspective (includes CTA at the end)
https://youtu.be/qS9rq_Sfiug

- Jean's video - TinaCMS 🦙 elevator pitch - Data in GitHub ? YES PLEASE 💦🤤
https://youtube.com/shorts/1OwMuAsTThw

Add a link to your video for upvotes !

<img width="500" alt="SCR-20240530-lrwv" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/38869720/f4eb1e76-a7cb-4f8d-96ef-e26d3186eb3f">

**Figure: Next time you're in this situation, you'll be prepared 🦙**

**Comment by JeanThirion** [comment]
Gert but I'm a groupie 🥰
@gertcmarx-hub 

My video: https://youtube.com/shorts/1OwMuAsTThw?feature=share

**Reply by Marxoz** [reply]
🥰

That is an old account btw, from my previous life.  @Marxoz is how I roll now :)

**Comment by PothieuG** [comment]
What is great is that TinaCMS is a great and user friendly tool that helps developers and non-developers working together !

My video : https://www.youtube.com/watch?v=B_aAcqV_qsA

**Comment by tkapa** [comment]
Of the 3 I think that Gert's video was awesome. Just maybe make sure the trucks aren't around next time 🤣

https://youtube.com/shorts/bAbeJNXCkuA?si=dVCr7AHSppREk_yu

**Comment by tiagov8** [comment]
I've already introduced Tina to a friend, but with way more time to explain it. It's good to practice a pitch and have a way to explain it concisely and simple for when you don't have much time.

My video: https://youtube.com/shorts/ufYMjbPDmCY?si=OvlOIGKiwPDz15pr

**Comment by tiagov8** [comment]
BTW - I don't think we should have 2 different rules for pitches... company and product... they seem too similar to justify being separated:
- https://www.ssw.com.au/rules/elevator-pitch/
- https://www.ssw.com.au/rules/pitch-a-product/

**Reply by jaydenalchin** [reply]
+1 - they're in the same category and definitely feel like duplicates

**Comment by MarcusIrmscherSSW** [comment]
Loved Gerts video it was concise, informative and personable.

**Comment by Marxoz** [comment]
3 different videos for 3 different audiences I think. 

I liked Piers' video as it was more developer focussed, and I think Tina is a product that you will probably sell to developers working on a big project rather than the client itself.

**Comment by camillars** [comment]
A pitch needs to be short and straight to the point, so my favourites are @piers-sinclair & @Marxoz 💫

my video - https://youtu.be/yy5_KQ2H5OA (ignore the background noise please 😆)

**Comment by Levijj22** [comment]
This was fun.
It forced you to be concise and made you think about what the most important feature of Tina is...which is different for everyone.

Some clients will be interested in the Editorial workflow, others the data.
That's why it's vital to ask questions when in Tina sales calls so you can concentrate on the features the client cares about.

**Comment by TobyChurches** [comment]
I liked all 3, but Gert's video was my favourite. I liked the brief overview (makes me want to look into it more)

My video: https://youtube.com/shorts/XzXPJZnj6tQ?si=7gjXaCRhy7LsOfwl

**Comment by GabbyWard** [comment]
This was a good exercise (especially for a rookie like myself!) I liked Pier's quick elevator pitch, felt like he got the points across pretty quick

I haven't been able to record my one yet, but I wonder if you can do like an "over-edited, speedy" style of youtube video (kind of like your typical YT shorts) which also shows TinaCMS in action + speaking vs only speaking to the points.

**Comment by jeoffreyfischer** [comment]
I like Adam's video because it gives context of where Tina sits comparted to the CMS

**Comment by leahy268** [comment]
I liked Adams video.  Mostly because I have had problems with a vendor wanting to extract lot's of money to get a copy of a clients data.  Being in GitHub and already owned by the client is a great thing.

**Comment by sethdaily** [comment]
Different perspectives from different people - all good depending on who the audience is. Tailor it to the person you're talking to

**Comment by Calinator444** [comment]
I personally prefer Piers' video. The delivery is excellent and he tells you everything you need to know about Tina in a short window. The only small nitpick I have is that there's some issues with the audio mix. Really gave my left ear a workout.


Anyway, here's a link to my shot at selling Tina

https://youtube.com/shorts/2YHoiiHD3l8

**Comment by alexleessw** [comment]
Liked Piers - short, sweet and to the point!

https://youtube.com/shorts/fbZju_QcIQ0

**Comment by Hona** [comment]
I liked Adam's more.

If you pitch too short, and you're pitching to the right person, you haven't got them on board.

If the person doesn't want to listen, 30 seconds wont matter vs 2 minutes.

Developers are passionate, so share the passion, and share how you can help solve their daily struggles.

With all that said, here's my 1 take wonder attempt at this

[![](https://i3.ytimg.com/vi/BrUIiLl8YZc/maxresdefault.jpg)](https://www.youtube.com/watch?v=BrUIiLl8YZc)

**Reply by Hona** [reply]
Don't forget to like, comment & subscribe

**Comment by AntPolkanov** [comment]
Liked Adam's videos as it clearly describes the pain.
See: https://youtu.be/dDq-8x386-Q

**Comment by jimmy1992abc** [comment]
I cannot believe I am recording this video in one shot.
See https://youtube.com/shorts/w0gttYEx9Lw

My little complaint:
* Adam's video is over 60 seconds, it is not legal!
* The Llama is not cute in red fur

**Comment by JackDevAU** [comment]
Always a good exercise to try and pitch an idea. As with Tiago's point I think we should merge the 2 rules on this topic together

**Comment by MattParkerDev** [comment]
A fun exercise.

**Comment by Ben0189** [comment]
Liked Gert's Video - sounds very natural and concise

**Comment by awaltos** [comment]
🥇 @piers-sinclair for the win - short, clear, confident, even threw in a cuss word 🤭

https://youtube.com/shorts/o2pmH2buZE0?feature=share
^ my attempt

Having pitch skills are important not only for plugging a product but can also be used to capture interest about anything whether it's you introducing yourself or explaining a concept or at the start of a presentation. Recording a video to plug a product is unnatural to me and I don't really feel comfortable doing it, however, being pushed out of one's comfort zone can be a good way to grow.

**Comment by YalalCakirSSW** [comment]
I like Piers' video. Clear, short and on point.

**Comment by amankumarrr** [comment]
I enjoy watching everyone's videos, and I particularly liked Jean's, Gert's, and Piers' videos—they really hit the spot.

**Comment by cvionnet** [comment]
Liked Adam's video as it highlights the importance of keeping control of our data and the problems associated with proprietary databases.

**Comment by matt-goldman-ssw** [comment]
Gert's video came across as authentic

**Comment by tombui99** [comment]
I like everyone's videos. All are very nice  👍

**Comment by chrisschultzssw** [comment]
Adam's video was my favourite, it explained the pain and how Tina solves the probllama.

**Comment by bradystroud** [comment]
This was fun - but my pitch would be different for every audience

This does not come naturally to me - normally I would ask questions to cater my pitch to the audience. 

E.g. I could explain TinaCMS to a tradie 🔨, but would need to know what the do first. (I'd say something like "imagine if clients could renovate their own houses, thats what TinaCMS enables")

https://youtube.com/shorts/P0DFb9uVDN4

**Comment by steven0x51** [comment]
Gert's video is my favourite

**Comment by ricksu978** [comment]
I like Piers' video because it's bloody awesome!

**Comment by tomek-i** [comment]
I prefer Adam's video as it sounds less like a pitch and more like a snipped from a casual conversation and hence more believable.
It also sounds like he believes what he is saying compared to the other videos which sound they are scripted and acted out.

**Comment by stephanfakossw** [comment]
Like Gert's video the best - friendly face...zippy but informative

https://youtube.com/shorts/Sxs0y5Z-RPQ?feature=share

**Comment by uly1** [comment]
https://youtube.com/shorts/UAGgBMUlFNA?si=QwgQtsR7E0izK93C

**Comment by PennyWalker** [comment]
I always enjoy an elevator pitch because I think it helps me learn about the things we're talking about. You have to be succinct and think about it before you action.

My video: https://youtube.com/shorts/9fkWCxzyEvk?si=_0zU1jCJhqR-n5IW

**Comment by bahjatalaadelssw** [comment]
Piers' video made it approachable.

My video: https://youtube.com/shorts/YC5K9VLOIZo

**Comment by jerryluossw** [comment]
I like Piers' video and Jean's video, Because they started by saying that Tina's data was stored on Github, that s is the most attractive.

**Comment by jimmidier** [comment]
All videos are awesome and I like Adam's video the best as the pain point was brought up and knowing how Tino solves the problem makes it super convincing

**Comment by duanxinhuan** [comment]
The only point of CMS to me is:
Boosting sales!!

**Comment by Aibono1225** [comment]
I like Adam's video for the history part with a clear comparison between Tina and other CMSs

**Comment by KristenHu** [comment]
I like Piers' video, cause he mentioned that Tina CMS is GitHub-based, that's the most unique part for Tina I think.

**Comment by tino-liu** [comment]
I enjoyed Adam's, although it is longer than 60 seconds, but his video really points out how Tina.io stands out.
My video: [https://youtu.be/KSeN_H6qAlc?si=oSYxROGRN7t4ms_x](https://youtu.be/KSeN_H6qAlc?si=oSYxROGRN7t4ms_x)
Problem is, it's hard to really come up with a good pitch without going a little in depth. Soooo, I my video went past 60 seconds.

**Comment by bettybondoc** [comment]
In the form, my original vote was for Piers' because it's straight to the point and gave me what I care most about right off the bat — owning my content. But in this GitHub discussion, Jean's was added and his was a little better. Haha.

My video - https://youtu.be/UEm385xESlQ

**Comment by christoment** [comment]
All videos are good - I like Adam's video since it has CTA at the end 😄

**Comment by danielmackay** [comment]
If I had to pick one video, it would be Piers's one.  Direct, to the point, natural sounding, and CTA at the end.

My video: https://youtu.be/rpT1uRDpV_c
